etcd/conf.yaml

```diff
@@ -104,6 +104,15 @@ eduid:
                 issuer: https://sandbox.orcid.org
             MONGO_URI: mongodb://eduid_orcid:eduid_orcid_pw@mongodb.eduid_dev
             AM_BROKER_URL: amqp://eduid:eduid_pw@rabbitmq/am
+        eidas:
+            SERVER_NAME: eidas.eduid.docker:8080
+            APPLICATION_ROOT: /
+            SECRET_KEY: supersecretkey
+            LOGGER_NAME: eidas
+            LOG_LEVEL: DEBUG
+            MONGO_URI: mongodb://eduid_eidas:eduid_eidas_pw@mongodb.eduid_dev
+            AM_BROKER_URL: amqp://eduid:eduid_pw@rabbitmq/am
+            MSG_BROKER_URL: amqp://eduid:eduid_pw@rabbitmq/msg
```

Comment:
There was yesterday a rearragement of settings in this file, and the SAML2_SETTINGS_MODULE was moved from the common to the authn sections. I noticed that eduid_webapp.eidas uses that setting, but yesterday there was no eidas section here, so I'm reminding you here that you need it explicitly :)

Reply:
Good reminder. I will fix that.